Kinvey.Core/Auth/Credential.cs

```diff
@@ -189,13 +197,14 @@ public static Credential From(KinveyAuthResponse response)
 		/// <param name="user">User.</param>
 		public static Credential From(User user)
 		{
-			return new Credential(user.Id, user.AccessToken, user.AuthToken, user.UserName, user.Attributes, user.Metadata, null, null, user.KinveyClient.DeviceID);
+			return new Credential(user.Id, user.AccessToken, user.AuthSocialID, user.AuthToken, user.UserName, user.Attributes, user.Metadata, null, null, user.KinveyClient.DeviceID);
 		}
 
 		public static Credential From(NativeCredential nc)
 		{
 			return new Credential(nc.UserID,
 			                      nc.Properties[Constants.STR_ACCESS_TOKEN],
+			                      null, // TODO add _socialIdentity object here
```

Comment:
@tejasranade This will be addressed as part of the fix for [MLIBZ-1479](https://kinvey.atlassian.net/browse/MLIBZ-1479)